files/scripts/asic_status.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+is_chassis_supervisor() {
+    if [ -f /etc/sonic/chassisdb.conf ]; then
```

Comment:
Need to find a different way to identify it is supervisor or linecard as this file /etc/sonic/chassisdb.conf is present in both sup and linecard

Reply:
@judyjoseph @abdosi 
Quick clarification - The check is correct. In database-chassis.service, that is what is used to start the chassis-database only on the Supervisor. Since the file is not present on the line-card, chassis-database does not start there.

```
[Unit]
Description=database-chassis container
Requires=docker.service
ConditionPathExists=/etc/sonic/chassisdb.conf
After=docker.service
After=config-chassisdb.service
Requires=config-chassisdb.service
StartLimitIntervalSec=1200
StartLimitBurst=3
```

Reply:
Ok, yes - there is a similar reference here https://github.com/Azure/sonic-buildimage/blob/c508a10c08636c78a3fb4d7454cc93769b8fbf92/files/image_config/config-chassisdb/config-chassisdb#L40.

So even though the file /usr/share/sonic/device/$platform/chassisdb.conf, is present in both supervisor and linecard, the variable "start_chassis_db=1" is set in supervisor which is used to copy the file to  /etc/sonic/chassisdb.conf

@mprabhu-nokia, @vganesan-nokia , @shyam77git, @anamehra -- Could you confirm this approach is followed in different chassis models ?

Reply:
@shyam77git, @anamehra can you confirm and sign-off these changes.

Reply:
At the outset, this seems OK i.e. having and checking chassisdb.conf presence on/for Supervisor.
 Anand is validating this further on modular chassis to see how it goes

Reply:
Instead of /etc/sonic/chassisdb.conf, may we have be some other mechanism like an entry in  DEVICE_METADATA to distinguish between Supervisor and other cards? Non VOQ chassis may not need a chassis db.

Reply:
As suggested in PR https://github.com/Azure/sonic-buildimage/pull/8065 (in review comments) we can have a variable like "supervisor=1" in platform_env.conf.

Reply:
Sure, once that gets committed, we can take it up as part of a bug fix PR. Don't want to block this PR with additional dependencies now. Let me know if that is ok to proceed.